dynamodb/src/main/scala/uk/gov/nationalarchives/DADynamoDBClient.scala

```diff
@@ -31,6 +31,30 @@ class DADynamoDBClient[F[_]: Async](dynamoDBClient: DynamoDbAsyncClient):
   extension [T](completableFuture: CompletableFuture[T])
     private def liftF: F[T] = Async[F].fromCompletableFuture(Async[F].pure(completableFuture))
 
+  /** Deletes a list of items with primary keys of type T from Dynamo. Unprocessed items will be retried
+    *
+    * @param tableName
+    *   The name of the table
+    * @param primaryKeys
```

Comment:
I'm assuming it's not just a list of primary keys,  right? i.e. it's not `List("1ee4d549-af7b-4054-9ab3-28f836c7f130", "588849b7-c7e4-4ba8-bde5-af4a4c349e15")` and instead it's more like key-value pairs? If so, should we maybe call it `primaryKeyAttributes`?

Reply:
Yeah, that makes sense, `primaryKeyAttributes` is good.